python/src/SobolIndicesAlgorithmImplementation_doc.i.in

```diff
@@ -117,47 +113,65 @@ We are now going to mix these two samples to get an estimate of the sensitivity
    \right)
 
 For the sake of stability, computations will be performed with centered output.
-Let :math:`G_0` be the empirical mean of all collected output samples and define:
+Let :math:`\overline{\vect{g}}` be the sample mean of samples 
+:math:`\vect{g}(\mat{A})` and :math:`\vect{g}(\mat{B})`.
+Let :math:`\tilde{\vect{g}}` be the empirically centered function defined by:
```

Comment:
Could you perhaps insert "for any :math:`\vect{x} \in \Rset^{n_X}`" between "defined" and "by"? If you do this, you also need to insert a point at the end of the mathematical expression at line 122.

Reply:
Done.